uwg_2023/uwg_2023_burst_demo.ipynb

```diff
@@ -0,0 +1,324 @@
+    "3. Because of advantages 1 and 2, HyP3 can offer larger quantities of useful data to users"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0881b00e-f430-48b7-9dd3-c82baae3adbf",
+   "metadata": {},
+   "source": [
+    "To get a sense for the migrating frame issue, take a look at these side-by-side SBAS searches over Mt. Edgecumbe. The left search was done using [SLCs in Vertex](https://search.asf.alaska.edu/#/?zoom=8.266&center=-134.416,56.456&polygon=POLYGON((-135.762%2057.0503,-135.7468%2057.0503,-135.7468%2057.0562,-135.762%2057.0562,-135.762%2057.0503))&flightDirs=Descending&resultsLoaded=true&granule=S1B_IW_SLC__1SDV_20211221T151602_20211221T151629_030125_0398E3_1C5A-SLC&dataset=SENTINEL-1&productTypes=SLC&beamModes=IW), and the right was done using [bursts in Vertex](https://search.asf.alaska.edu/#/?zoom=10.950&center=-135.912,56.947&polygon=POLYGON((-135.762%2057.0503,-135.7468%2057.0503,-135.7468%2057.0562,-135.762%2057.0562,-135.762%2057.0503))&flightDirs=Descending&resultsLoaded=true&granule=S1_372326_IW3_20211221T151616_VV_1C5A-BURST&dataset=SENTINEL-1%20BURSTS&beamModes=IW&polarizations=VV)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b3f7837f-0feb-4640-a2d6-439f9753ec84",
+   "metadata": {},
+   "source": [
+    "<img src=\"vertex.png\" alt=\"Search Comparison\" style=\"width:900px;\"/>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5330ccb4-488d-4288-9ac2-ec41b5ba0b45",
+   "metadata": {},
+   "source": [
+    "## 3. Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c98292cf-7721-464b-807f-33da6bd15712",
+   "metadata": {},
+   "source": [
+    "Follow the developer setup instructions in the [HyP3-ISCE2 readme](https://github.com/asfhyp3/hyp3-isce2#developer-setup), then run the command below:\n",
+    "\n",
+    "```bash\n",
+    "mamba activate hyp3-isce2\n",
+    "mamba install -y jupyter jupyterlab hyp3_sdk\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a02761b-0384-4490-aa93-69ba5f8a7544",
+   "metadata": {},
+   "source": [
+    "## 4. Local Python Package Interface"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8bb6bdb9-baac-4220-8bc0-d5cbe88c2f38",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir local_run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9c9b8424-7d38-4667-8831-8913b1a2a9f8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cd local_run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f7e43296-a265-4be3-aeec-f82e05695da5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -m hyp3_isce2 ++process insar_tops_burst \\\n",
+    "  S1_136231_IW2_20200604T022312_VV_7C85-BURST \\\n",
+    "  S1_136231_IW2_20200616T022313_VV_5D11-BURST \\\n",
+    "  --looks 20x4"
```

Comment:
Installing hyp3-isce2 registers an entrypoint for insar_tops_burst; I prefer using it directly:
```suggestion
    "!insar_tops_burst S1_136231_IW2_20200604T022312_VV_7C85-BURST  S1_136231_IW2_20200616T022313_VV_5D11-BURST --looks 20x4"
```